include/tfrt/host_context/async_value.h

```diff
@@ -349,7 +350,7 @@ class AsyncValue {
   const bool is_refcounted_ : 1;
 
   // Unused padding bits.
-  unsigned unused_ : 4;
+  // unsigned unused_ : 4;
```

Comment:
Yes, although I should really remove the line instead of commenting it out.

Long version: the original code is not portable - bit fields packing is implementation defined, and some compilers (notably MSVC) will not pack together bitfields with different base types.